tests/test_models/test_catboost.py

```diff
@@ -144,3 +144,79 @@ def test_save_load(model, example_tsds):
     horizon = 3
     transforms = [LagTransform(in_column="target", lags=list(range(horizon, horizon + 3)))]
     assert_model_equals_loaded_original(model=model, ts=example_tsds, transforms=transforms, horizon=horizon)
+
+
+@pytest.fixture()
+def ar_dataset_w_exog():
```

Comment:
We already have fixture with regressors `example_reg_tsds`

Reply:
This fixture is not quite suitable as it has only one regressor. Will use another one.